assets/css/style.css

```diff
@@ -371,7 +380,7 @@ th{
 
 /* fixed size for navbar icons */
 .cmt-widget #cmt .btn-viewer-icn .glyphicon {
-    font-size: 24px;
+    font-size: 22px;
```

Comment:
This will change it for all implementations, not sure if we would want that...

Reply:
Yes, if you look at the icons when the measurement tool is open, it looks larger than usual so I made it slightly smaller on purpose and I think it still looks good for all implementations.